index.Rmd

```diff
@@ -42,6 +43,7 @@ roads<-read_sf("_data/1877_roads_talbot.shp")%>%
   st_transform(crs=4326)
 
 #This is the houses shapefile from the historic map and Census Data
+#
```

Comment:
You probably just haven't gotten to this step yet but don't forget to clean the houses data and remove any NAs, and any race that is not White or Black (unless you wanted to include Mulatto).